spec/integration/working_file_path_spec.rb

```diff
@@ -198,7 +198,25 @@
   end
 
   context "without Settings.matterhorn.media_path set" do
+
+    let(:media_path) { Dir.mktmpdir }
+
+    around(:example) do |example|
+      begin
+        # explicitly unset path; don't assume unset in config
+        old_media_path = Rails.application.secrets.matterhorn_client_media_path
+        Rails.application.secrets.matterhorn_client_media_path = ''
+
+        example.run
+
+        Rails.application.secrets.matterhorn_client_media_path = old_media_path
+      ensure
+        FileUtils.remove_entry media_path
+      end
+    end
+
     it 'returns blank' do
+      expect(Rails.application.secrets.matterhorn_client_media_path).to be_blank
```

Comment:
Testing against local settings, the above code remove upstream assumption that the media path is unset. Purpose, allow tests to work against settings used locally.